docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+docker build -t codersrank_extractor:latest .
```

Comment:
I created the CodersRank organization on Docker Hub you can upload the image there. 
I think there should be another script to build the docker image and to run it.
something like: `build-docker.sh` and `run-docker.sh`

Reply:
Done